src/routes/brands/updateBrand.ts

```diff
@@ -70,46 +72,8 @@ const schema = {
   body: BrandSchema,
   response: {
     200: BrandSchema,
-    404: {
-      title: "InvalidBrand",
-      description: "Invalid or missing Brand data.",
-      type: "object",
-      required: ["error"],
-      properties: {
-        error: {
-          type: "object",
-          required: ["code", "message"],
-          properties: {
-            code: {
-              type: "string",
-            },
-            message: {
-              type: "string",
-            },
-          },
-        },
-      },
-    },
-    500: {
-      title: "Error",
-      description: "An unknown error occurred when trying to update brands.",
-      type: "object",
-      required: ["error"],
-      properties: {
-        error: {
-          type: "object",
-          required: ["code", "message"],
-          properties: {
-            code: {
-              type: "string",
-            },
-            message: {
-              type: "string",
-            },
-          },
-        },
-      },
-    },
+    404: ErrorSchemaJson,
```

Comment:
in this case, we don't want to change the entire object, Only the error at line 79.